SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -56,7 +56,7 @@ NodeProxy : BusPlug {
 			if(freeGroup) {
 				oldGroup = group;
 				group = nil;
-				bundle.sched((fadeTime ? this.fadeTime) + (server.latency ? 0), { oldGroup.free });
+				server.sendBundle((fadeTime ? this.fadeTime) + (server.latency ? 0), [11, oldGroup.nodeID]);
```

Comment:
`/n_free`. It's in Server Command Reference. In terms of cargo cult programming, I wonder if it is more efficient to do:

`PushInt 11`

instead of

`PushLiteral Symbol 'n_free'`

If not, I'd prefer to use `n_free`, but otherwise, I'd add a comment.